### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by SilvaPaula** [tpook92/HaploBlocker#6]
Hi, I'm trying to install HaploBlocker but I get the following error. I tried also to install it directly using devtools::install_github('tpook92/HaploBlocker', subdir='pkg') but same error. Also when trying to load RandomFieldsUtils.

Any help will be much appreciated.

> install.packages("~/Downloads/HaploBlocker_1.5.8.tar", repos = NULL, type='source')
* installing *source* package ‘HaploBlocker’ ...
** using staged installation
** libs
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/RandomFieldsUtils/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include -fopenmp -fPIC  -Wall -g -O2  -c chaploblocker.cc -o chaploblocker.o
clang: error: unsupported option '-fopenmp'
make: *** [chaploblocker.o] Error 1
ERROR: compilation failed for package ‘HaploBlocker’
* removing ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library/HaploBlocker’

**Comment by tpook92** [comment]
I just checked and had no problems on all of my machines (Linux + Windows). I do not have a Mac to check sadly.

In case you are working on a windows machine it might be necessary to use the .zip-version of the package (https://github.com/tpook92/HaploBlocker/blob/master/HaploBlocker_1.5.8.zip) or prior to using install Rtools (https://cran.r-project.org/bin/windows/Rtools/).

It the issue is still occuring after this i would need more information on your system to check the issue more thoroughly.

**Comment by SilvaPaula** [comment]
Yeah, I am using Mac. For what I have been searching is a Mac issue with Rtools.
Thanks for your reply.
